charts/kkr/inapp/values.yaml

```
@@ -0,0 +1,96 @@
+# Default values for inapp.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: 293315485843.dkr.ecr.ap-northeast-2.amazonaws.com/kkr/inapp
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ''
+
+config:
+  apiBaseUrl: ''
+  useVConsole: ''
+
+secret:
+  oauth:
+    authBaseUrl: ''
+    clientId: ''
+    clientSecret: ''
+
+imagePullSecrets: []
+nameOverride: ''
+fullnameOverride: ''
+
+podAnnotations: {}
+
+podSecurityContext:
+  {}
+  # fsGroup: 2000
+
+securityContext:
+  {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+responseTransformer:
+  enabled: false
+  config: {}
+
+httpsRedirect:
+  enabled: false
+
+
+ipRestriction:
+  enabled: false
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: false
+  className: ''
+  annotations:
+    kubernetes.io/ingress.class: kong
+    kubernetes.io/tls-acme: "true"
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+    konghq.com/plugins: goerli-alpha-frontend-https-redirect
```

```suggestion
    # konghq.com/plugins: 
```
plugin은 디폴트에서 제거해도 되겠습니다.